sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -156,8 +163,8 @@
     },
     "/configurations/testQueries": {
       "post": {
-        "summary": "Validates the target condition query and custom metric queries for a configuration.",
-        "description": "Validates the target condition query and custom metric queries for a configuration. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "summary": "Validates the target condition and custom metric queries for a configuration.",
```

Comment:
As Tim pointed out in the previous PR, summary tags are not being used anywhere in the autogen code; might be worth deleting them entirely.

Reply:
Yes I agree. I will delete for now. Do you guys know if there is any guidance to keep them at all even if they are redundant? My only thought is that the service team will generate this so we may just get them back based on the logic there.